### PR TITLE
Ulyanov2016instance Summary 1

### PR DESCRIPTION
**Pull request by rll2396**

### CODE REVIEW

File: _summaries/ulyanov2016instance_1.md [hunk 1]
```diff
@@ -1,19 +1,64 @@
...
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* Core Idea
+
+    Instance normalization can help a feed-forward generator network achieve better performance at stylizing images.
```
Comment (aabayomi) [inline _summaries/ulyanov2016instance_1.md:13]: I think you can also a brief description of what the problem Instance normalization is solving.

File: _summaries/ulyanov2016instance_1.md [hunk 2]
```diff
@@ -1,19 +1,64 @@
...
+
+* Context
+    * Gatys et al. (2016) showed promising results for using CNNs for the task of image stylization
+
+    * The authors (2016) previously showed that the same task can be achieved hundreds of times faster at a cost to the quality of the images generated
+
+    * In this paper they show that changing the normalization from batch to instance can improve the quality
```
Comment (aabayomi) [inline _summaries/ulyanov2016instance_1.md:20]: For context is there a reason the author has decided to change the normalization? is it because the computational efficiency ? 
I think adding some pointers to the history of the problem will make a case for the good results listed here.

File: _summaries/ulyanov2016instance_1.md [hunk 3]
```diff
@@ -1,19 +1,64 @@
...
+
+    ![Contrast](ulyanov2016instance_1c.png)
+
+    Here are the stylized images. The top row is with batch normalization. The bottom row is with instance normalization.
+
+    ![Contrast](ulyanov2016instance_1d.png)
 
 ## TL;DR
-* Three
-* Bullets
```
Comment (aabayomi) [inline _summaries/ulyanov2016instance_1.md:18]: I think you can also add some more points e.g like the shortcomings of the generator networks.

**Review by aabayomi — COMMENTED** [review]
Overall 
I think it is a good summary. easy to read and understand. 
Additional notes on variant used in the experiment may be good to have.

**Comment by rll2396** [comment]
I've revised the summary. I didn't see the paper mention experimenting with any variants, so I don't think I can add that section. Thanks for the feedback. Let me know if you have any comments on revised summary. Otherwise, I'm satisfied with it as the final draft.
